src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -339,20 +417,32 @@ public void MonitoringSurvivedProcessMemorySize()
         [Fact]
         public void MonitoringTotalAllocatedMemorySize()
         {
-            Assert.Throws<InvalidOperationException>(() => { var t = AppDomain.CurrentDomain.MonitoringTotalAllocatedMemorySize; } );
+            RemoteInvoke(() => {
+                Assert.Throws<InvalidOperationException>(() => {
+                    var t = AppDomain.CurrentDomain.MonitoringTotalAllocatedMemorySize;
+                });
+                return SuccessExitCode;
+            }).Dispose();
         }
 
         [Fact]
         public void MonitoringTotalProcessorTime()
         {
-            Assert.Throws<InvalidOperationException>(() => { var t = AppDomain.CurrentDomain.MonitoringTotalProcessorTime; } );
+            Assert.Throws<InvalidOperationException>(() => {
+                var t = AppDomain.CurrentDomain.MonitoringTotalProcessorTime;
+            } );
         }
 
 #pragma warning disable 618
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
```

Comment:
I left it unchanged. Was disabled before and after this change for no reason. I only added reasons where I could get quickly into a conclusion. I did not want to spend too much time on unrelated changes.